src/utils/SafeTransferLib.sol

```diff
@@ -7,6 +7,19 @@ import {ERC20} from "../tokens/ERC20.sol";
 /// @author Solmate (https://github.com/Rari-Capital/solmate/blob/main/src/utils/SafeTransferLib.sol)
 /// @dev Caution! This library won't check that a token has code, responsibility is delegated to the caller.
 library SafeTransferLib {
+    
+    /*//////////////////////////////////////////////////////////////
+                              CUSTOM ERRORS
+    //////////////////////////////////////////////////////////////*/
+
+    error ETHTransferFailed();
+
+    error TransferFailed();
+
+    error TransferFromFailed();
```

Comment:
Swap the order of these two.

`TransferFromFailed()` comes before `TransferFailed()` in the code.